cider/tests/functionality/CiderStringTest.cpp

```diff
@@ -255,19 +255,150 @@ class CiderStringTestArrow : public CiderTestBase {
     create_ddl_ =
         R"(CREATE TABLE test(col_1 INTEGER NOT NULL, col_2 VARCHAR(10) NOT NULL);)";
 
-    input_ = {std::make_shared<CiderBatch>(QueryDataGenerator::generateBatchByTypes(
-        10,
+    QueryArrowDataGenerator::generateBatchByTypes(
+        schema_,
+        array_,
+        30,
         {"col_1", "col_2"},
-        {CREATE_SUBSTRAIT_TYPE(I32), CREATE_SUBSTRAIT_TYPE(Varchar)}))};
+        {CREATE_SUBSTRAIT_TYPE(I32), CREATE_SUBSTRAIT_TYPE(Varchar)},
+        {0, 0},
+        GeneratePattern::Sequence,
+        0,
+        10);
+  }
+};
+
+class CiderStringNullableTestArrow : public CiderTestBase {
+ public:
+  CiderStringNullableTestArrow() {
+    table_name_ = "test";
+    create_ddl_ = R"(CREATE TABLE test(col_1 INTEGER , col_2 VARCHAR(10) );)";
+
+    QueryArrowDataGenerator::generateBatchByTypes(
+        schema_,
+        array_,
+        30,
+        {"col_1", "col_2"},
+        {CREATE_SUBSTRAIT_TYPE(I32), CREATE_SUBSTRAIT_TYPE(Varchar)},
+        {2, 2},
+        GeneratePattern::Sequence,
+        0,
+        10);
```

Comment:
why function call's code style with one argument one line?
I think the following code more better:
```c++
QueryArrowDataGenerator::generateBatchByTypes(
    schema_, array_, 30, {"col_1", "col_2"},
    {CREATE_SUBSTRAIT_TYPE(I32), CREATE_SUBSTRAIT_TYPE(Varchar)},
    {2, 2}, GeneratePattern::Sequence, 0, 10);
```

Reply:
formatter will do this.